CHANGES.rst

```diff
@@ -3,6 +3,9 @@ Release Notes
 
 1.7 (unreleased)
```

Comment:
Because you're on an older version 1.7 is the only thing listed. We're actually working on 1.7.2 right now, which is the current unreleased version. 

Once you merge with master you'll likely have some conflicts to resolve. In that process you should make sure that you move the changelog changes up to the 1.7.2 block.